curriculum/challenges/english/10-coding-interview-prep/rosetta-code/sha-256.md

```diff
@@ -8,11 +8,12 @@ dashedName: sha-256
 
 # --description--
 
-**[SHA-256](https://en.wikipedia.org/wiki/SHA-256)** is the recommended stronger alternative to [SHA-1](https://rosettacode.org/wiki/SHA-1).
+The `SHA-2` family is a stronger alternative to `SHA-1`. The main difference between them is the length of the hash.
+Meaning `SHA-1` provides a shorter code with fewer possibilities for unique combinations. `SHA-2` or `SHA-256` creates a longer and thus more complex hash with more possibilities.
 
 # --instructions--
 
-Write a function that takes a string as a parameter and returns its SHA-256 digest.
+Research implemenation details and write a function that takes a string as the parameter and make it return a hash using `SHA-256`
```

Comment:
```suggestion
Research implemenation details and write a function that takes a string as the parameter and returns a hash using `SHA-256`
```

I think we should stick with the "function... takes... returns..." syntax here. Thoughts?

Reply:
Seems good to me 👍